cpp/recognition/cograph/BretscherCorneilHabibPaulCographRecognition.cpp

```diff
@@ -0,0 +1,212 @@
+/*
+ * BretscherCorneilHabibPaulCographRecognition.cpp
+ *
+ *  Created on: 2024
+ *      Author: fixikmila
+ */
+// Copyright 2024 milana
```

Comment:
If you leave the upper comment, you can get rid of this line

Reply:
fixed